packages/cmrjs/package.json

```diff
@@ -40,7 +40,7 @@
     "@cumulus/errors": "13.3.2",
     "@cumulus/launchpad-auth": "13.3.2",
     "@cumulus/logger": "13.3.2",
-    "got": "^11.8.1",
+    "got": "^11.8.5",
```

Comment:
So I was having trouble with my bootstrap and saw an error similar to what you saw with `keyv`. I eventually got it to work by _only_ upgrading npm though. I reverted the change here on my branch and used `"got": "^11.8.1",`, which worked fine. Might be worth re-trying.

I think "^11.8.1" and "^11.8.5" will also functionally work the same? They'll both pull the latest patch release on the minor version?

Reply:
It's the same, and 11.8.5 is the latest version for 11.8.x